src/applications/vaos/referral-appointments/hooks/useGetProviderById.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react';
```

Comment:
I like the hook. But we should probably update everything that fetches provider to use it as well